src/lib/commandBarConfigs/modelingCommandConfig.ts

```diff
@@ -330,6 +332,13 @@ export const modelingMachineCommandConfig: StateMachineCommandSetConfig<
         required: true,
         skip: true,
       },
+      axis: {
+        required: true,
+        inputType: 'selection',
+        selectionTypes: ['segment', 'sweepEdge', 'edgeCutEdge'],
+        multiple: false,
+        validation: revolveAxisValidator,
```

Comment:
Happy to see this! Will close https://github.com/KittyCAD/modeling-app/issues/4661

Reply:
And we definitely need to expand that to other things. Seems like the perfect way to prevent cylinder wall shell attempts